### DISCUSSION TITLE
Raspberry Pi 4 64-bit opencv installation

### DISCUSSION THREAD

**Discussion by aizeeland** [raspberrypi/picamera2#20] [Ideas]
The use of the Raspberry Pi 4 with an 64-bit installation and the latest opencv is described on this website:(https://qengineering.eu/install-opencv-4.5-on-raspberry-64-os.html)

Following the instructions to install the latest version of opencv along the instructions of picamera2 does not seem to give any problems for me. I used the installation script provided by the website. Maybe it is possible to use these installation steps instead of the pip3 install command for the installation of opencv? On the website it is also mentioned as a bad idea to use pip install for opencv because the pip install versions are not up-to-date and at the moment not 64-bit.

**Comment by davidplowman** [comment]
Thanks for this information. I've changed the instructions to install OpenCV from apt now, it seems more reliable!
